app/models/contributor.rb

```diff
@@ -0,0 +1,4 @@
+class Contributor < ActiveRecord::Base
+  has_many :suggested_councillors, through: :councillor_contributions
```

Comment:
Is this association necessary? If we're not using it we should remove that line.